test/utils.h

```diff
@@ -78,6 +78,7 @@ inline TensorView* makeConcreteTensor(
 inline TensorView* makeContigConcreteTensor(
     std::vector<int64_t> shape,
     DataType dtype = DataType::Float) {
+  c10::cuda::CUDACachingAllocator::emptyCache();
```

Comment:
Why is this?

Reply:
Thanks for catching. That was a leftover from an earlier try. Removed.

Reply:
It is still running locally but so far it looks similar.

Reply:
Tests are done and `device_stats.reserved_bytes[0].peak` reports same peak as before removal of this line.